src/main/java/bowling/domain/bowl/Gutter.java

```diff
@@ -1,20 +1,17 @@
 package bowling.domain.bowl;
 
-import bowling.domain.bowl.type.BowlTypeDto;
 import bowling.domain.pitch.PitchResult;
 import bowling.domain.score.Score;
-
-import java.util.List;
+import bowling.domain.score.Scores;
 
 public class Gutter extends Ended{
```

Comment:
```suggestion
public class Gutter extends Ended {
```

클래스 이름 뒤에 space가 있는 것이 컨벤션
다수의 클래스 뒤에 space가 없다.